libkbfs/reporter_kbpki.go

```diff
@@ -282,9 +269,17 @@ func (r *ReporterKBPKI) send(ctx context.Context) {
 			if !ok {
 				return
 			}
-			if suppressed {
-				stagedNotification = notification
-			} else if err := r.config.KeybaseService().Notify(ctx,
+			nt := notification.NotificationType
+			// Only these notifications are used in frontend:
+			// https://github.com/keybase/client/blob/0d63795105f64289ba4ef20fbefe56aad91bc7e9/shared/util/kbfs-notifications.js#L142-L154
+			if nt != keybase1.FSNotificationType_REKEYING &&
```

Comment:
This can still get pretty spammy :(

Reply:
Fortunately they don't happen often, and are going away in the future. So I'm inclined to just leave it here rather than coming up some deduplication logic. (I assume just sending the latest one wouldn't be the same.) WDYT?